layouts/partials/json-schema/resolve-additional-types.html

```diff
@@ -142,20 +129,17 @@
     {{ $this_object := partial "json-schema/resolve-allof" .this_object }}
 
     {{ $more_objects := partial "json-schema/resolve-additional-types" (dict "schema" $this_object "anchor_base" .anchor_base "name" $name) }}
-    {{/*
-      As far as I know we don't have something like Array.concat(), so add them one at a time
-    */}}
```

Comment:
I removed this comment, since I found it misleading. We *do* have something like `Array.concat` -- it's called `append`, which does concatenate arrays if you pass an array as its parameter. However, it's not very useful here, because we want to call `clean-object` on each item before adding it to `$all_objects`.